NN_chatCmd.js

```diff
@@ -16,6 +16,21 @@ nerthus.chatCmd.run = function(ch)
             log("["+ch.k+"] " + ch.n + " -> " + ch.t) //gdzie kto co
             return callback(ch)
         }
+        return false
+    }
+    return false
+}
+
+nerthus.chatCmd.run_ni = function(e)
+{
+    if (e[1].s !== "abs" && e[1].s !== "") return;
+    let clone = JSON.parse(JSON.stringify(e[1]))
+    let ch = nerthus.chatCmd.run(clone)
+    if(ch) {
```

Comment:
Co to jest robione z tym eventem?

Reply:
To tak:
e[0] to obiekty DOM wiadomości (mają dziwną strukturę)
e[1] to wiadomość tak jak wygląda na SI
Dlatego klona jej przepuszczam przez już napisany run. On zwraca mi wartość jeżeli jest coś, a jeżeli nie to zwraca false (dodałem to)
Jak przepuszczałem obiekt a nie klona to potem paskudnego dziwnego buga miałem co kilka godzin dochodziłem o co z tym chodzi

Następnie ładnie edytuje obiekty DOM wiadomości, która ma swoją specyficzną budowę, wstawiając nazwy klas czy też wiadomości w miejsce tych starych.

Na NI wiadomości czatu edytuje się właśnie za pomocą danych referencji na obiekty DOM.